onnxruntime/core/mlas/lib/sqnbitgemm_kernel_avx2.cpp

```diff
@@ -29,6 +29,47 @@ Module Name:
 #include "sqnbitgemm_m1_sym_kernel_avx2_int8_blklen32.h"
 #include "sqnbitgemm_m1_sym_kernel_avx2_int8_blklen64.h"
 
+void ConvertFp16ToFp32Avx(const MLAS_FP16* src_fp16, float* dst_fp32, uint64_t size)
```

Comment:
there is an existing implementation for this: MlasConvertHalfToFloatBuffer in MLAS.h

Reply:
consolidated.